src/GitHub.Exports.Reactive/ViewModels/GitHubPane/IPullRequestReviewFileCommentViewModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Reactive;
+using ReactiveUI;
+
+namespace GitHub.ViewModels.GitHubPane
+{
+    /// <summary>
+    /// Represents a view model for a file comment in an <see cref="IPullRequestReviewViewModel"/>.
+    /// </summary>
+    public interface IPullRequestReviewFileCommentViewModel
+    {
+        /// <summary>
+        /// Gets the body of the comment.
+        /// </summary>
+        string Body { get; }
+
+        /// <summary>
+        /// Gets the path to the file, relative to the root of the repository.
+        /// </summary>
+        string RelativePath { get; }
+
+        /// <summary>
+        /// Gets a comment which opens the comment in a diff view.
```

Comment:
Should this be simply, "Opens the comment in a diff view"?

Reply:
Ah, should be "Gets a command...". If you look around at C# documentation, properties always start with "Gets a" (for read-only properties) or "Gets or sets a" (for read/write properties). See https://softwareengineering.stackexchange.com/questions/193244/is-the-gets-or-sets-necessary-in-xml-documentation-of-properties and http://stylecop.soyuz5.com/SA1623.html.

Reply:
Thanks, I hadn't noticed that convention.